torchvision/ops/diou_loss.py

```diff
@@ -0,0 +1,86 @@
+import torch
+
+from ..utils import _log_api_usage_once
+from .boxes import _upcast
+
+
```

Comment:
Commenting above the file might be better. As comments in the function call will be a small execution of commented code everytime we call code? (Is there a subtle performance difference? Not sure but always had this doubt)